### PR TITLE
Initial config of Travis CI

### PR DESCRIPTION
**Pull request by boggy-d**

### CODE REVIEW

**Review by AndreiUngur — DISMISSED** [review]
LGTM, keeping in mind that the `exit 0` on the tests is currently only for testing purposes & will be switched back to an `exit 1` in the future (to make sure tests don't silently fail)!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+node_js:
+  - lts/*
+  - node
+
+cache: npm
+
+script:
+  - cd neutralino.js
+  - npm test
```
Comment (erickzhao) [inline .travis.yml]: nit: add newline at EOF
```suggestion
  - npm test
  
```

File: neutralino.js/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "echo \"Error: no test specified\" && exit 0",
```
Comment (erickzhao) [inline neutralino.js/package.json:7]: nit: should not say `Error` and exit with code `0`.
```suggestion
    "test": "echo \"No test specified\" && exit 0",
```

**Review by erickzhao — DISMISSED** [review]
Few small comments, but LGTM overall.

File: neutralino.js/package.json (hunk 2 above)
Comment (boggy-d) [reply neutralino.js/package.json:7]: This is getting removed in the next PR right after this, so can we merge it knowing this

**Review by erickzhao — APPROVED** [review]

**Review by kraglalbert — APPROVED** [review]

**Pull request merged**